scripts/qa_crowbarsetup.sh

```diff
@@ -3261,23 +3228,15 @@ function set_proposalvars
         deployceph=
     fi
     # C3: Cloud5 only has ceph for SLES12
-    if iscloudver 5 && [ -z "$want_sles12" ] ; then
+    if iscloudver 5 ; then
```

Comment:
so testing upgrades form SOC5 to 6 is no more important, because all customers migrated already?